### PR TITLE
Rfc2119

### PR DESCRIPTION
**Pull request by dericed**
Here are a few places where I thought a different rfc2119 term was
intended. Review carefully!

Intended to resolve https://github.com/privatezero/flac_markdown/issues/40

### CODE REVIEW

File: flac.md [hunk 1]
```diff
@@ -105,27 +109,27 @@ Before the formal description of the stream, an overview might be helpful.
...
+  - The blocksize bits in the `FRAME_HEADER` (see [`FRAME_HEADER` section](#frameheader)) MUST be 0001-1110. The blocksize MUST be <= 16384; if the sample rate is <= 48000 Hz, the blocksize MUST be <= 4608.
+  - The sample rate bits in the `FRAME_HEADER` MUST be 0001-1110.
+  - The bits-per-sample bits in the `FRAME_HEADER` MUST be 001-111.
+  - If the sample rate is <= 48000 Hz, the filter order in `LPC subframes` (see [`SUBFRAME_LPC` section](#subframelpc)) MUST be less than or equal to 12, i.e. the subframe type bits in the `SUBFRAME_HEADER` (see [`SUBFRAME_HEADER` section](#subframeheader)) SHOULD NOT be 101100-111111.
```
Comment (retokromer) [inline flac.md]: The values in the previous 4 lines should begin with `0b`, because they are binary.

File: flac.md [hunk 2]
```diff
@@ -225,38 +229,38 @@ Data     | Description
...
 `u(1)`            | `1` if the CUESHEET corresponds to a Compact Disc, else `0`.
-`u(7+258\*8)`     | Reserved. All bits must be set to zero.
-`u(8)`            | The number of tracks. Must be at least 1 (because of the requisite lead-out track). For CD-DA, this number must be no more than 100 (99 regular tracks and one lead-out track).
-`CUESHEET_TRACK`+ | One or more tracks. A CUESHEET block is required to have a lead-out track; it is always the last track in the CUESHEET. For CD-DA, the lead-out track number must be 170 as specified by the Red Book, otherwise is must be 255.
+`u(7+258\*8)`     | Reserved. All bits MUST be set to zero.
+`u(8)`            | The number of tracks. Must be at least 1 (because of the requisite lead-out track). For CD-DA, this number MUST be no more than 100 (99 regular tracks and one lead-out track).
+`CUESHEET_TRACK`+ | One or more tracks. A CUESHEET block is REQUIRED to have a lead-out track; it is always the last track in the CUESHEET. For CD-DA, the lead-out track number MUST be 170 as specified by the Red Book, otherwise is MUST be 255.
```
Comment (privatezero) [inline flac.md]: looks like there was a typo here in the original spec that has held over. Should be 'otherwise it MUST'

File: flac.md (hunk 1 above)
Comment (dericed) [inline flac.md]: out of scope with the intent of the PR

File: flac.md (hunk 2 above)
Comment (dericed) [inline flac.md]: fixed and rebased

**Comment by dericed** [comment]
This PR is updated. Please re-review.

File: flac.md (hunk 1 above)
Comment (retokromer) [inline flac.md]: OK. It was just to avoid other conflicts. I will open an _ad hoc_ PR.

**Comment by retokromer** [comment]
Indeed, I tried four times to fix the few missed `0b` by NN, since the very first correction of them, which has also been done twice, by NN and by me. BTW, in my opinion, e.g. `0b00-0b11` would be better than the current `0b00 - 0b11` (with spaces before and after the dash). If you agree, I will fix that. 

I don’t know why – nor when – the order of the section has been modified.

**Pull request merged**